src/test/java/bio/terra/service/filedata/azure/AzureIngestFileConnectedTest.java

```diff
@@ -141,14 +146,20 @@ public void setup() throws Exception {
 
   @After
   public void cleanup() throws Exception {
+    try {
+      tableDao.deleteFileMetadata(fileId, billingProfile, storageAccountResource);
+      tableDirectoryDao.deleteDirectoryEntry(tableServiceClient, fileId);
+    } catch (Exception ex) {
+      logger.error("Unable to clean up metadata for fileId {}", fileId, ex);
```

Comment:
I wanted to add this just so that something doesn't silently fail. I don't necessarily think we'll manually clean up, but if someone is looking into the test, then they might notice that it's not working as expected.

Reply:
If this is unexpected, then maybe the test should fail and the exception shouldn't be caught. If the test doesn't fail, then it seems unlikely anyone will look at the test logs.

Reply:
The only case where this would fail is if the test itself failed. I wanted the try/catch around these methods so that the rest of the cleanup methods would still run, but we'd still know that it didn't all clean up successfully. (Which, hopefully should be a helpful log when debugging the failure)

Thanks for thinking through this so throughly. I'm good with removing the try/catch, but I think the extra info on debugging could be helpful.

Reply:
No need to change it, thanks for the explanation!